aemanalyser-maven-plugin/src/main/java/com/adobe/aem/analyser/mojos/AemAnalyseMojo.java

```diff
@@ -237,10 +242,10 @@ public void execute() throws MojoExecutionException, MojoFailureException {
             return;
         }
 
-        final VersionUtil versionUtil = new VersionUtil(this.getLog(), this.project, 
-                this.artifactHandlerManager, this.artifactMetadataSource, 
-                this.project.getRemoteArtifactRepositories(), this.project.getPluginArtifactRepositories(), 
-                this.mavenSession.getLocalRepository(), this.mavenSession.isOffline());
+        final VersionUtil versionUtil = new VersionUtil(this.getLog(), this.project, artifactHandlerManager,
+                this.repoSystem, this.repoSession,
+                this.remoteProjectRepositories, this.remotePluginRepositories,
```

Comment:
Why can't we use project.getXXX ? This avoids the additional parameters

Reply:
Good point, done in https://github.com/adobe/aemanalyser-maven-plugin/pull/128/commits/f525d21c5fd4971bbd17370e1e9a01f7def77c30